pkg/cmd/server/kubernetes/node.go

```diff
@@ -144,42 +142,12 @@ func (c *NodeConfig) RunKubelet() {
 	}
 
 	cadvisorInterface, err := cadvisor.New(4194)
-	if err == nil {
-		// TODO: use VersionInfo after the next rebase
-		_, err = cadvisorInterface.MachineInfo()
-	}
 	if err != nil {
-		glog.Errorf("WARNING: cAdvisor cannot be started: %v", err)
-		cadvisorInterface = &cadvisor.Fake{}
+		glog.Fatalf("Error instantiating cadvisor: %v", err)
```

Comment:
It does when I cross-compile to linux natively on the mac. I'm trying with a binary built with the openshift-release container to see if it makes a difference. (Just much slower)